.github/workflows/integration.yml

```diff
@@ -43,7 +42,7 @@ jobs:
           DEBUG: 'transloadit:*'
 
       - name: Generate the badge from the json-summary
-        run: node test/generate-coverage-badge.js coverage/coverage-summary.json
+        run: npx tsx test/generate-coverage-badge.ts coverage/coverage-summary.json
```

Comment:
Better use yarn?

Reply:
I don't think yarn has an equivalent of `npx`. there's `yarn exec` but then we'd have to first `yarn add tsx` into the node-sdk repo, not sure we want that?

Reply:
It can be a devDependency? Its nice to pin these things anyway, our CI does rely on it after all